packages/@ama-sdk/schematics/schematics/migrate/index.ts

```diff
@@ -0,0 +1,35 @@
+import { Rule, type Tree } from '@angular-devkit/schematics';
+import { MigrateSchematicsSchemaOptions } from './schema';
+import { getMigrationRuleRunner, type MigrationRulesMap } from '@o3r/schematics';
+import { resolve } from 'node:path';
+
+const tsMigrationMap: MigrationRulesMap = {
+
+};
+
+const isTypescriptSdk = (tree: Tree) => {
+  return tree.exists('/tsconfig.json');
+};
+
+/**
+ * Facilitate the migration of a version to another by the run of migration rules
+ * @param options
+ */
+function migrateFn(options: MigrateSchematicsSchemaOptions): Rule {
+
+  const to: string = options.to || JSON.parse(require(resolve(__dirname, '..', '..', 'package.json'))).version;
```

Comment:
should we log something if the user tries a version higher than the one installed?